### PR TITLE
Add support for connected accounts in checkout session

### PR DESCRIPTION
**Pull request by nerder**
Closes https://github.com/flutter-stripe/flutter_stripe/issues/893 and #683

### CODE REVIEW

File: packages/stripe_checkout/lib/src/platforms/stripe_checkout.dart [hunk 1]
```diff
@@ -149,11 +155,18 @@ class _CheckoutPageState extends State<CheckoutPage> {
...
+    var options = '';
+    if (stripeAccountId != null) {
+      options += 'stripeAccount: "$stripeAccountId"';
+    }
+
+    final redirectToCheckoutJs = '''
+      var stripe = Stripe("$publishableKey", {$options});
+      stripe.redirectToCheckout({sessionId: "$sessionId"});
+    ''';
+    print(redirectToCheckoutJs);
```
Comment (remonh87) [inline packages/stripe_checkout/lib/src/platforms/stripe_checkout.dart:169]: let's remove the print :)

**Review by remonh87 — APPROVED** [review]
I think this looks @jonasbark  @jamesblasco  can one of you have a look as well?

File: packages/stripe_checkout/lib/src/platforms/stripe_checkout.dart (hunk 1 above)
Comment (nerder) [reply packages/stripe_checkout/lib/src/platforms/stripe_checkout.dart]: Fixed :)

**Comment by nerder** [comment]
Hey, there is any action missing from my side in order to get his merged?

**Pull request merged**